CHANGELOG.md

```diff
@@ -2,14 +2,19 @@
 
 All notable, unreleased changes to this project will be documented in this file. For the released changes, please visit the [Releases](https://github.com/mirumee/saleor/releases) page.
 
-## Unreleased
+# 3.5.0 [Unreleased]
+
+...
+
```

Comment:
It seems like there is a lack of `# 3.4.0` label.

Reply:
Good catch, fixed.